install/bootstrap_livecd_remote_iso.md

```diff
@@ -519,39 +547,52 @@ Some files are needed for generating the configuration payload. See the [Command
       >    ```
 
-   1. Continue with the next step to [prepare site init](#prepare-site-init).
+   1. Continue to the next step to [verify and backup `system_config.yaml`](#verify-csi-versions-match).
 
-<a name="prepare-site-init"></a>
-### 4.2 Prepare Site Init
+<a name="verify-csi-versions-match"></a>
+### 4.2 Verify and Backup `system_config.yaml`
 
-1. First, prepare a shim to facilitate going through the site-init guide:
+1. Verify that the newly generated `system_config.yaml` matches the current version of CSI.
 
-    ```bash
-    pit# mkdir -vp /mnt/pitdata &&
-         mount -v --bind /var/www/ephemeral/ /mnt/pitdata/
-    ```
+   1. View the new `system_config.yaml` file and note the CSI version reported near the end of the file.
 
-1. Follow the procedures to [Prepare Site Init](prepare_site_init.md) directory for your system.
+      ```bash
+      pit# cat $PITDATA/prep/${SYSTEM_NAME}/system_config.yaml
+      ```
 
-1. Remove the shim:
+   1. Note the version reported by the `csi` tool.
 
-    > This uses `rmdir` to safely remove the directory, preventing accidental removal if one does not notice a `umount` command failure.
+      ```bash
+      pit# csi version
+      ```
 
-    ```bash
-    pit# cd ~ && umount -v /mnt/pitdata/ && rmdir -v /mnt/pitdata
-    ```
+   1. The two versions should match. If they do not, determine the cause and regenerate the file.
+
+1. Copy the new `system_config.yaml` file somewhere safe to facilitate re-installs.
+
+1. Continue to the next step to [prepare site init](#prepare-site-init).
+
+<a name="prepare-site-init"></a>
+### 4.3 Prepare Site Init
+
+> **Important:** Although the command prompts in this procedure are `linux#`, the procedure should be
+> performed on the PIT node.
+
+Prepare the `site-init` directory by performing the [Prepare Site Init](prepare_site_init.md) procedures.
 
 <a name="bring---up-the-pit-services-and-validate-pit-health"></a>
 ## 5. Bring-up the PIT Services and Validate PIT Health
 
 1. Set the same variables from the `csi config init` step from earlier, and then invoke `pit-init.sh` to set up the PIT server for deploying NCNs.
-   > **`Note:`** `pit-init.sh` will re-run `csi config init`, copy all generated files into place, apply the CA patch, and finally restart daemons. This will also re-print the `metalid.sh` content in case it was skipped in the previous step. **Re-installs** can skip running `csi config init` entirely and simply run `pit-init.sh` after gathering CSI input files into `/var/www/ephemeral/prep`.
+   > **Note:** `pit-init.sh` will re-run `csi config init`, copy all generated files into place, apply the CA patch, and finally restart daemons. This will also re-print the `metalid.sh` content in case it was skipped in the earlier step. **Re-installs** can skip running `csi config init` entirely and simply run `pit-init.sh` after gathering CSI input files into `$PITDATA/prep`.
```

Comment:
I don't think it is actually true that `csi config init` can be skipped as the generated `customizations.yaml` output of `csi config init` is required for the prepare_site_init.md procedure which occurs before `pit-init.sh` is run.

Also unless [CASMINST-3260](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-3260) was fixed then `pit-init.sh` should *NOT* run `csi config init` as it risks generating a SLS file that has different IPs for the WLM services than the ones in the `customizations.yaml` file generated during prepare_site_init.md. I'll do some digging and open a new ticket for this if necessary.

Reply:
Good catch!

Reply:
I thought that too when I read this. I think maybe @rustydb added this section in a recent PR. I know that @kimjensen-hpe tried doing this on a fanta install and hit problems, so I do agree that doesn't seem to be true.

I'll add another commit to this PR to remove the language which suggests this path.